spec/system/admin_customize_form_templates_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+describe "Admin Customize Form Templates", type: :system, js: true do
+  let(:form_template_page) { PageObjects::Pages::FormTemplate.new }
+  let(:ace_editor) { PageObjects::Components::AceEditor.new }
+  fab!(:admin) { Fabricate(:admin) }
+  fab!(:form_template) { Fabricate(:form_template) }
+
+  before do
+    SiteSetting.experimental_form_templates = true
+    sign_in(admin)
+  end
+
+  describe "when visiting the page to customize form templates" do
+    it "should show the existing form templates in a table" do
+      visit("/admin/customize/form-templates")
+      expect(form_template_page).to have_form_template_table
+      expect(form_template_page).to have_form_template(form_template.name)
+    end
+
+    it "should show the form template structure in a modal" do
+      visit("/admin/customize/form-templates")
+      form_template_page.click_view_form_template
+      expect(form_template_page).to have_template_structure("some yaml template: value")
+    end
+  end
+
+  describe "when visiting the page to edit a form template" do
+    it "should prefill form data" do
+      visit("/admin/customize/form-templates/#{form_template.id}")
+      expect(form_template_page).to have_name_value(form_template.name)
+      # difficult to test the ace editor content (todo later)
```

Comment:
@keegangeorge can you please open a minor PR to add your name to this TODO? We should try not to add TODO comments to the codebase without doing this since otherwise it has no ownership (I see you already did it for other TODOs in this PR which is great :+1: )